microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/definition/AbstractAnnotationDefinitionParticipant.java

```diff
@@ -47,18 +48,35 @@
 
 	private final String annotationName;
 
-	private final String annotationMemberName;
+	private final String[] annotationMemberNames;
+
+	protected Function<String, String> propertyReplacer;

```

Comment:
`private final Function<String, String> propertyReplacer;`

and defines a propertyReplacer if you need toaccess to the property replacer in the class which extends this class.

Reply:
I would need protected for subclass access correct?

Reply:
Oh sorry, you mean defin getPropertyReplacer?